cmd/operator-opamp-bridge/agent/testdata/agent.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: ws://127.0.0.1:4320/v1/opamp
+protocol: wss
+capabilities:
+  - Unspecified
+  - ReportsStatus
```

Comment:
Just an FYI: ReportsStatus [is always set](https://github.com/open-telemetry/opamp-spec/blob/79771f7fff1e1ae10ccdaffeee5b2b3689c8dd8c/proto/opamp.proto#LL522C10-L522C10) by OpAMP client, so it doesn't matter if the user specifies it here or no.
I don't think you need to do anything special about it, just wanted to make sure you are aware and avoid listing it e.g. in the operator docs in the future.